pkg/admissioncontroller/webhook/auth/seed/graph/eventhandler_shoot.go

```diff
@@ -145,6 +146,18 @@ func (g *graph) handleShootCreateOrUpdate(ctx context.Context, shoot *gardencore
 			configMapVertex := g.getOrCreateVertex(VertexTypeConfigMap, shoot.Namespace, shoot.Spec.Kubernetes.KubeAPIServer.StructuredAuthentication.ConfigMapName)
 			g.addEdge(configMapVertex, shootVertex)
 		}
+
+		if len(v1beta1helper.GetShootAuthorizationConfigurationConfigMapName(kubeAPIServer)) > 0 {
+			configMapVertex := g.getOrCreateVertex(VertexTypeConfigMap, shoot.Namespace, shoot.Spec.Kubernetes.KubeAPIServer.StructuredAuthorization.ConfigMapName)
```

Comment:
```suggestion
if configMapName := v1beta1helper.GetShootAuthorizationConfigurationConfigMapName(kubeAPIServer); len(configMapName) > 0 {
			configMapVertex := g.getOrCreateVertex(VertexTypeConfigMap, shoot.Namespace, configMapName)
```

Nit: Slightly more stable to use the helpers output.